### PR TITLE
Add 'build lzo' action

### PR DESCRIPTION
**Pull request by andrewdmcleod**
The libs are owned by root, this might need to be changed - pending testing

### CODE REVIEW

File: actions/build-lzo [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+        pass
+    check_call(['tar', '-cvzf', 'lzo-dist.tar.gz', 'share/', 'lib/'])
+    produced_file = os.path.join(builder.workspace, 'lzo-dist.tar.gz')
+    
+    # {} will be replace with the first digits of the digest
+    digest, final_file = builder.set_output_file(produced_file)
+    return digest, final_file
+
+
+def inplace_change(filename, old_string, new_string):
```
Comment (ktsakalozos) [inline actions/build-lzo]: This is not used anywhere, right?

File: actions/build-lzo [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+    builder.checkout_source(LZO_GIT)
+
+
+def build_binary(builder):
+    hookenv.log("Building lzo libraries")
+    build_command = ['mvn', 'clean', 'package']
+    builder.build_binary(build_command)
+    os.makedirs(builder.workspace + '/share/hadoop/common/lib')
+    os.makedirs(builder.workspace + '/lib/native')
+    copyfile(builder.workspace + '/target/hadoop-lzo-' + LZO_VERSION + '.jar', builder.workspace + '/share/hadoop/common/lib/hadoop-lzo-' + LZO_VERSION + '.jar')
```
Comment (ktsakalozos) [inline actions/build-lzo]: The output directory has a "lzo" (builder.workspace + '/lzo/target/hadoop-lzo-'........ )

2016-05-12 09:12:55 INFO build-lzo Traceback (most recent call last):
2016-05-12 09:12:55 INFO build-lzo   File "/var/lib/juju/agents/unit-gobblin-binary-builder-0/charm/actions/build-lzo", line 84, in <module>
2016-05-12 09:12:55 INFO build-lzo     build_binary(builder)
2016-05-12 09:12:55 INFO build-lzo   File "/var/lib/juju/agents/unit-gobblin-binary-builder-0/charm/actions/build-lzo", line 39, in build_binary
2016-05-12 09:12:55 INFO build-lzo     copyfile(builder.workspace + '/target/hadoop-lzo-' + LZO_VERSION + '.jar', builder.workspace + '/share/hadoop/common/lib/hadoop-lzo-' + LZO_VERSION + '.jar')
2016-05-12 09:12:55 INFO build-lzo   File "/usr/lib/python3.4/shutil.py", line 108, in copyfile
2016-05-12 09:12:55 INFO build-lzo     with open(src, 'rb') as fsrc:
2016-05-12 09:12:55 INFO build-lzo FileNotFoundError: [Errno 2] No such file or directory: '/tmp/workspace/target/hadoop-lzo-0.4.20-SNAPSHOT.jar'
^

File: actions/build-lzo (hunk 1 above)
Comment (andrewdmcleod) [inline actions/build-lzo]: Nope, just copied it straight out of build-gobblin, i'll remove it

File: actions/build-lzo (hunk 2 above)
Comment (andrewdmcleod) [inline actions/build-lzo]: Must have made this change on the unit and forgot to implement in code

File: actions/build-lzo [hunk 3]
```diff
@@ -36,7 +36,7 @@ def build_binary(builder):
     builder.build_binary(build_command)
     os.makedirs(builder.workspace + '/share/hadoop/common/lib')
     os.makedirs(builder.workspace + '/lib/native')
-    copyfile(builder.workspace + '/target/hadoop-lzo-' + LZO_VERSION + '.jar', builder.workspace + '/share/hadoop/common/lib/hadoop-lzo-' + LZO_VERSION + '.jar')
+    copyfile(builder.workspace + '/target/hadoop-lzo-' + LZO_VERSION + '.jar', builder.workspace + '/lzo/share/hadoop/common/lib/hadoop-lzo-' + LZO_VERSION + '.jar')
```
Comment (ktsakalozos) [inline actions/build-lzo]: It should be:
copyfile(builder.workspace + '/lzo/target/hadoop-lzo-' + LZO_VERSION + '.jar', ..............

The lzo missing is on the location where the jar is created

**Comment by ktsakalozos** [comment]
Tested. LGTM. Merging it

**Pull request merged**